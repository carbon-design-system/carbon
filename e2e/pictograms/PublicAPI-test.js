/**
 * Copyright IBM Corp. 2018, 2018
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @jest-environment node
 */

'use strict';

const CarbonPictograms = require('@carbon/pictograms');

describe('@carbon/pictograms', () => {
  it('should not update exports without a semver change', () => {
    expect(Object.keys(CarbonPictograms).sort()).toMatchSnapshot();
  });
});
