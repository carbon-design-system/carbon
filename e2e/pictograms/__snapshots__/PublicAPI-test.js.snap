// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@carbon/pictograms should not update exports without a semver change 1`] = `
Array [
  "AcceleratedComputing",
  "AcceleratingTransformation",
  "AccessManagement",
  "ActiveServer",
  "AddDevice",
  "AddDocument",
  "AdvancedFraudProtection",
  "AdvancedThreats",
  "Advocate",
  "AdvocateMask",
  "Agility",
  "AgilityWithHybridMulticloud",
  "Agriculture",
  "Ai",
  "AiEthics",
  "AiExplainability",
  "AiPrivacy",
  "AiReady",
  "AiRobustness",
  "AiTransparency",
  "AiTrust",
  "AirConditioner",
  "Airplane",
  "Alarm",
  "AlchemyDataNews",
  "AlchemyLanguage",
  "AlchemyVision",
  "Americas",
  "AmsterdamCanal",
  "AmsterdamFarm",
  "AmsterdamWindmill",
  "Analyze",
  "AnalyzesData",
  "AnalyzingContainers",
  "AnonymousUsers",
  "Apartment",
  "Api",
  "AppDeveloper",
  "AppModernization",
  "Apple",
  "Application",
  "ApplicationSecurity",
  "Apps",
  "Archive",
  "ArgentinaObelisk",
  "ArtTools_01",
  "AsiaAustralia",
  "AssessmentUsed",
  "AssetManagement",
  "Assets",
  "Atlanta",
  "AudioData",
  "AuditTrail",
  "AugmentedReality",
  "Austin",
  "AutomateModularManagement",
  "Automated",
  "AutomationDecision",
  "Automobile",
  "Availability",
  "BabyBottle",
  "Backpack",
  "Badge",
  "Bag",
  "Balloon",
  "BalloonHotAir",
  "Bangalore",
  "Banking",
  "Barcelona",
  "Bee",
  "BeijingMunicipal",
  "BeijingTower",
  "BerlinBrandenburgGate",
  "BerlinCathedral",
  "BerlinTower",
  "Bicycle",
  "BigData",
  "BirthdayCake",
  "Blender",
  "Blockchain",
  "Bluepages",
  "BostonZakimBridge",
  "BoxPlot",
  "Broom",
  "Budapest",
  "BudapestCitadella",
  "BudapestLabSkyline",
  "BugVirusMalware",
  "Build",
  "BuildAndDeployPipeline",
  "BuildApplicationsAnywhere",
  "BuildLeadershipAndCulture",
  "Building",
  "Bulldozer",
  "Bus",
  "BusinessContinuity",
  "BusinessContinuity_02",
  "CLanguage",
  "CPlusPlusLanguage",
  "Cafe",
  "CairoGizaPlateau",
  "Calendar",
  "CalendarDate",
  "CalendarEvent",
  "Camera",
  "CanadaMapleLeaf",
  "Candy",
  "Capitol",
  "CarbonForIbmDotcom",
  "CarbonForIbmProduct",
  "Care",
  "CargoCrane",
  "CargoShip",
  "Cell",
  "ChartArea",
  "ChartBar",
  "ChartBubble",
  "ChartBubbleLine",
  "ChartCandlestick",
  "ChartCustom",
  "ChartErrorBar",
  "ChartEvaluation",
  "ChartHighLow",
  "ChartHistogram",
  "ChartLine",
  "ChartMultiLine",
  "ChartMultiType",
  "ChartParallel",
  "ChartPie",
  "ChartQuadrantPlot",
  "ChartRadar",
  "ChartRiver",
  "ChartScatterplot",
  "ChartScreePlot",
  "ChartStepper",
  "ChartSunburst",
  "ChartTSne",
  "ChartTwoYAxis",
  "Chart_3D",
  "Cheese",
  "Cherries",
  "Chicago",
  "ChileEasterIsland",
  "ChileHandOfTheDesert",
  "ChipCircuit",
  "ChipCredit",
  "ChipDebit",
  "ChooseHowToGetStarted",
  "CirclePacking",
  "ClientFinancing_01",
  "ClientFinancing_02",
  "ClothesRack_01",
  "ClothesRack_02",
  "CloudAnalytics",
  "CloudAssets",
  "CloudBuilderProfessionalServices",
  "CloudComputing",
  "CloudDataServices",
  "CloudDownload",
  "CloudEcosystem",
  "CloudGuidelines",
  "CloudInfrastructureManagement",
  "CloudManagedServices",
  "CloudNative_01",
  "CloudNative_02",
  "CloudNative_03",
  "CloudOracle",
  "CloudPakFamily",
  "CloudPakForApplications",
  "CloudPakForBusinessAutomation",
  "CloudPakForData",
  "CloudPakForIntegration",
  "CloudPakForMulticloudManagement",
  "CloudPakForNetworkAutomation",
  "CloudPakForSecurity",
  "CloudPakSystem",
  "CloudPartners",
  "CloudPlanning",
  "CloudPlatform",
  "CloudSap",
  "CloudServices",
  "CloudServicesPricing",
  "CloudStorage",
  "CloudStrategy",
  "CloudTutorials",
  "CloudUpload",
  "CloudVmware",
  "Cloudy",
  "CloudyDewy",
  "CloudyHazy",
  "CloudyHumid",
  "CloudyPartial",
  "CloudyWindy",
  "Cluster",
  "CoatHanger",
  "CobolLanguage",
  "Code",
  "CodeSyntax",
  "CognosAnalytics",
  "CollaborateWithTeams",
  "Collaboration",
  "College",
  "ColombiaCathedralOfLasLajas",
  "ColorContrast",
  "CommercialFinancing_01",
  "CommercialFinancing_02",
  "Compliant",
  "CompostBin",
  "ConceptExpansion",
  "ConceptInsights",
  "ConditionBuilder",
  "ConfidentialComputing",
  "Connect",
  "ConnectApplications",
  "ConnectToCloud",
  "ConnectedDevices",
  "ConnectedEcosystem",
  "ConnectedNodesToTheCloud",
  "Connectivity",
  "Console",
  "ConsoleWireless",
  "Construct",
  "ConstructionWorker",
  "ConstructionWorkerMask",
  "ConsumerEngagementFoodJourney",
  "Container",
  "ContainerMicroservices",
  "ContainerizedApplications",
  "Containers",
  "ContainersAndCloudNative",
  "ContentDesign",
  "Continuous",
  "ContinuousSecurity",
  "Contract",
  "Control",
  "ControlPanel",
  "ControlsFramework",
  "Conversation",
  "Cookie",
  "CopenhagenPlanetarium",
  "CopenhagenSnekkja",
  "Coronavirus",
  "Coupon",
  "CreditCard",
  "Crop",
  "Cupcake",
  "CurveCubic",
  "CurveExponential",
  "CurveInverse",
  "CurveLinear",
  "CurveLogarithmic",
  "CurveLogistic",
  "CurvePower",
  "CurveQuadratic",
  "CustomReports",
  "CustomWorkloads",
  "CustomerService",
  "Customizable",
  "Dashboard",
  "DataApis",
  "DataBackup",
  "DataCenters",
  "DataInsights",
  "DataPrivacy",
  "DataPrivacyKey",
  "DataPrivacy_02",
  "DataProcessing",
  "DataProtectionDataSecurity",
  "DataSecurity",
  "DataSet",
  "DataStorage",
  "DataTransfer",
  "DecisionVelocity",
  "DeepLearning",
  "Delete",
  "DeliverInsights",
  "Delivered",
  "DeliveryTruck",
  "DeployingContainers",
  "Deployment",
  "DesignAndDevelopment_01",
  "DesignAndDevelopment_02",
  "DesignLeadership",
  "DesignResearch",
  "DesignThinkingTeam",
  "Desktop",
  "DetectAndStopAdvancingThreats",
  "DeveloperTools",
  "DeveloperZOs",
  "DevicePairing",
  "DevicesAtIbm",
  "Devops",
  "DevopsToolchain",
  "Dialogue",
  "Digital",
  "DigitalId",
  "DigitalTrust",
  "Dining",
  "DistributionLedger",
  "Dna",
  "DoNot",
  "DoNot_02",
  "Docker",
  "Doctor",
  "DoctorPatient",
  "DocumentConversion",
  "DocumentSecurity",
  "Documentation",
  "DoorHandle",
  "Download_01",
  "Download_02",
  "DragAndDropInterface",
  "Dropper",
  "DubaiPalmIslands",
  "DubaiSkyscraper",
  "DublinBrewery",
  "DublinCastle",
  "DuplicateFile",
  "DynamicWorkloads",
  "Earth",
  "EaseOfUse",
  "Ecosystem",
  "EcuadorQuito",
  "Edge",
  "Education",
  "Efficient",
  "Electric",
  "ElectricCar",
  "ElectricCharge",
  "ElementsOfTheCloud",
  "Elevator",
  "Embed",
  "EmergencyExit",
  "EmployeeInsights",
  "Encryption",
  "EndpointProtection",
  "EnergyCrisis",
  "Engine",
  "EnterpriseDesignThinking_01",
  "EnterpriseDesignThinking_02",
  "Envelope",
  "ErlenmeyerFlask",
  "EscalatorDown",
  "EscalatorUp",
  "EsgReporting_01",
  "EsgReporting_02",
  "EuropeAfrica",
  "EventAutomation",
  "EventDriven",
  "EventEndpointManagement",
  "EventProcessing",
  "EventStreams",
  "ExpandHorz",
  "ExpandUser",
  "ExpandVert",
  "Expansion",
  "Export_01",
  "Export_02",
  "ExtendTheDataCenter",
  "Extensible",
  "ExtractText",
  "Eye",
  "EyewashStation",
  "FaceDissatisfied",
  "FaceNeutral",
  "FaceSatisfied",
  "FaceVeryDissatisfied",
  "FaceVerySatisfied",
  "Factory",
  "Fairness",
  "Farm_01",
  "Farm_02",
  "Farmer_01",
  "Farmer_02",
  "Fast",
  "FasterInnovationWithPartners",
  "Faucet",
  "FaultTolerant",
  "Feedback_01",
  "Feedback_02",
  "FileBackup",
  "FileTransfer",
  "FilterAndGroupData",
  "FilterVariable",
  "FinanceStrategy",
  "FinancialConsultant",
  "FinancialGain",
  "FinancialNetworks",
  "FinancialNews",
  "FireAlarm",
  "FireExtinguisher",
  "Firecracker",
  "Firefighter",
  "Firewall",
  "FirstAid",
  "Flag",
  "FlashStorage",
  "FlashingContent",
  "Flexibility",
  "Flexibility_02",
  "FlexibleCompute",
  "FlexibleInfrastructure",
  "FlowChart",
  "FlowChartDetail",
  "Flower_01",
  "Flower_02",
  "FocusOnCode",
  "Fog",
  "Folder",
  "Forecasting",
  "Forklift",
  "Fragile",
  "FreeTrial",
  "Fuel",
  "FullyManaged",
  "Gear",
  "GeographicFlexibility",
  "GetAheadOfRiskAndCompliance",
  "Gift",
  "GlobalAnalytics",
  "GlobalAssetsAndRecovery_01",
  "GlobalAssetsAndRecovery_02",
  "GlobalBusinessServices",
  "GlobalCurrency",
  "GlobalExchange",
  "GlobalFinanceEuro",
  "GlobalFinanceNetwork",
  "GlobalFinanceSterling",
  "GlobalFootprint",
  "GlobalMarkets",
  "GlobalMarketsBar",
  "GlobalNetwork",
  "GlobalPartner",
  "GlobalSecurity",
  "GlobalStrategy",
  "GlobalTechnologyServices",
  "Globe",
  "GlobeLocations",
  "GoLanguage",
  "Goals",
  "GovernUsersAndIdentities",
  "Government_01",
  "Government_02",
  "GraphicIntensiveWorkloads",
  "GreenEnergy",
  "GreenIt_01",
  "GreenIt_02",
  "Group",
  "Growth",
  "Guitar",
  "Hail",
  "HailHeavy",
  "HailMixed",
  "HamburgPhilharmone",
  "Handicap",
  "HandicapActive",
  "Handshake",
  "HardDrive",
  "HardDriveNetwork",
  "HardIceCream",
  "Hazy",
  "Headphones",
  "Heart",
  "HeartHealth",
  "HeatMap_01",
  "HeatMap_02",
  "HelpDesk",
  "HighFive",
  "HighPerformance",
  "HighRiskUsers",
  "HighVolumeData",
  "HighlyAvailable",
  "Hills",
  "HomeFront",
  "HomeGarage",
  "HomeProfile",
  "HongKong",
  "HongKongCityscape",
  "Hospital",
  "Hpi",
  "Humid",
  "Hurricane",
  "HybridCloud",
  "HybridCloudInfrastructure",
  "HybridCloudServices",
  "HybridCloud_02",
  "IbmAutomationPlatform",
  "IbmCloud",
  "IbmIx",
  "IbmRpa",
  "IbmZ",
  "IbmZ16",
  "IbmZ16MultiFrame",
  "IbmZ16PlusCloud",
  "IbmZ16SingleFrame",
  "IbmZAndLinuxoneMultiFrame",
  "IbmZAndLinuxoneSingleFrame",
  "IbmZPartition",
  "IdBadge",
  "Idea",
  "Ideate",
  "IdentifyAndAccess",
  "IdentifyAndResolveIssues",
  "IdentityTrustAssessment",
  "IndiaSouthAsia",
  "InfrastructureSecurity",
  "InnerSource",
  "Innovate",
  "Insights",
  "InspectData",
  "Insurance",
  "Integration",
  "Integration_02",
  "Intelligence",
  "Intercom",
  "Invoice",
  "IotMunich",
  "ItInfrastructureSoftware",
  "Java",
  "Javascript",
  "Juice",
  "Justice",
  "KeepDry",
  "KeepYourOwnKey",
  "KeyUsers",
  "Keyboard",
  "KnowsDarkData",
  "KualaLumpur",
  "Kubernetes",
  "KubernetesPod",
  "LanguageTranslation",
  "Language_01",
  "Language_02",
  "Language_03",
  "Language_04",
  "Language_05",
  "Lantern",
  "Launch",
  "Leader",
  "LiftAndShift",
  "Lightning",
  "Link",
  "ListBullet",
  "ListCheckbox",
  "Location",
  "Lock_01",
  "Lock_02",
  "LockedNetwork_01",
  "LockedNetwork_02",
  "London",
  "LondonBigBen",
  "Love",
  "LoweringRisk",
  "Luggage",
  "Lungs",
  "MachineLearning_01",
  "MachineLearning_02",
  "MachineLearning_03",
  "MachineLearning_04",
  "MachineLearning_05",
  "MachineLearning_06",
  "MachineLearning_07",
  "MadridCathedral",
  "MadridSkyscrapers",
  "MadridStatue",
  "Magnify",
  "MailVerse",
  "ManageApplicationsAnywhere",
  "ManagingContainers",
  "ManagingContractualFlow",
  "ManagingItems",
  "Marketplace",
  "Mas",
  "MasterThreatHunting",
  "MathCurve",
  "Maximize",
  "Medal_01",
  "Medal_02",
  "Medical",
  "MedicalCharts",
  "MedicalStaff",
  "Melbourne",
  "Meter",
  "MexicoCityAngelOfIndependence",
  "MexicoCityMuseoSoumaya",
  "Microorganisms",
  "Microphone",
  "Microscope",
  "MilanDuomoDiMilano",
  "MilanSkyscrapers",
  "Minimize",
  "MobileAdd",
  "MobileChat",
  "MobileDevices",
  "MobilePhone",
  "Modernize",
  "Monitor",
  "MonitoredItemOnConveyor",
  "MoonCake",
  "MoonFull",
  "MortarAndPestle",
  "Moscow",
  "MovementInOverlappingNetworks",
  "MovementOfGoods_01",
  "MovementOfGoods_02",
  "MovementOfGoods_03",
  "MovementOfItems",
  "MovingDolly",
  "Mqa",
  "Mri",
  "MriPatient",
  "Multitask",
  "Munich",
  "Music",
  "NaturalLanguageClassifier",
  "NaturalLanguageUnderstanding",
  "Network",
  "NetworkOfDevices",
  "NetworkProtection",
  "NetworkServices",
  "NetworkTraffic",
  "Networking_01",
  "Networking_02",
  "Networking_03",
  "Networking_04",
  "Networking_05",
  "Networking_06",
  "NewFinancialCustomerExperiences",
  "NewRevenueStreams",
  "Nice",
  "NightClear",
  "NoSmoking",
  "Notifications",
  "NycBrooklyn",
  "NycChryslerBuilding",
  "NycManhattan_01",
  "NycManhattan_02",
  "NycStatueOfLiberty",
  "NycWorldTradeCenter",
  "Office",
  "OilPump",
  "OilRig",
  "Okinawa",
  "OnPremise",
  "OnPremiseToCloud",
  "OpenSource",
  "OperateOffline",
  "OperatingSystem",
  "OperationalEfficiency",
  "OperationalMetrics",
  "Optimize",
  "OptimizeCashFlow_01",
  "OptimizeCashFlow_02",
  "Organization",
  "Ornament",
  "Overcast",
  "Overview",
  "PaperClip",
  "ParisArcDeTriomphe",
  "ParisLouvre",
  "ParisNotreDame",
  "ParisPompidouCenter",
  "Parliament",
  "PartnerRelationship",
  "Partnership",
  "Path",
  "Pattern",
  "PayForWhatYouUse",
  "Perfume",
  "Person_01",
  "Person_02",
  "Person_03",
  "Person_04",
  "Person_05",
  "Person_06",
  "Person_07",
  "Person_08",
  "Person_09",
  "PersonalityInsights",
  "PeruMachuPicchu",
  "PetriCulture",
  "PhpLanguage",
  "PillBottle_01",
  "Pills",
  "PlanningAnalytics",
  "PlayerFlow",
  "PliLanguage",
  "Podcast",
  "Police",
  "PopulationDiagram",
  "PortlandBuilding",
  "Power",
  "PragueCharlesBridgeTower",
  "PragueDancingHouse_01",
  "PragueDancingHouse_02",
  "Predictability",
  "PredictiveAnalytics",
  "Pregnant",
  "Prescription",
  "Presentation",
  "Presenter",
  "Price",
  "Printer",
  "PrivateNetwork_01",
  "PrivateNetwork_02",
  "PrivateNetwork_03",
  "PrivateNetwork_04",
  "Process",
  "Productivity",
  "ProfessionalMarketplace",
  "Progress",
  "ProtectCriticalAssets",
  "ProvenTechnology",
  "PublicCloudToPrivateCloud",
  "Puzzle",
  "QQPlot",
  "QrCode",
  "Quantum",
  "QuantumComputing",
  "QuantumSafe",
  "Question",
  "QuestionAndAnswer",
  "RLanguage",
  "Racetrack",
  "Rainy",
  "RainyHeavy",
  "RaleighNc",
  "RandomSamples",
  "Rank",
  "ReactToData",
  "ReadOnly",
  "RealTime",
  "Receipt",
  "Recycle",
  "RecycleBin",
  "RedHatApplications",
  "RedefiningFinancialServices",
  "ReducingCost",
  "Refinery",
  "Refresh",
  "RelationshipDiagram",
  "RelationshipExtraction",
  "Reliability",
  "Reliability_02",
  "Renew",
  "RenewTeam",
  "Repeat",
  "Report",
  "Research",
  "Reset",
  "ResetHybridCloud",
  "ResetSettings",
  "Resilience",
  "ResourceHealth",
  "Resources",
  "RetailSustainable",
  "RetrieveAndRank",
  "RichTextFormat",
  "RioDeJaneiro",
  "Robot",
  "Robotics",
  "RockOn",
  "Rocket",
  "Rome",
  "RotateDevice",
  "SaasEnablement",
  "SalesConnect",
  "SampleFile",
  "SanFrancisco",
  "SanFranciscoFog",
  "SaoPaulo",
  "SapHana",
  "Satellite",
  "SatelliteDish",
  "SaveTime",
  "Scale",
  "Scale_02",
  "ScalingContainers",
  "ScanCode",
  "ScatterMatrix",
  "Seattle",
  "SecureData",
  "SecureHybridCloud",
  "SecureProfile",
  "SecureSearch",
  "Security",
  "SecurityAsAService",
  "SecurityHygiene",
  "SecurityIntelligence",
  "SecurityShield",
  "Security_02",
  "SelectProduct",
  "SelectRange",
  "SeoulGyeongbokgungPalace",
  "ServerOperatingSystems",
  "ServerRack",
  "Serverless",
  "Servers",
  "ShanghaiCityscape",
  "ShanghaiOrientalPearlTower",
  "SharingData",
  "Shirt",
  "Shop",
  "ShoppingCart",
  "Shower",
  "Singapore",
  "SingleSignOn",
  "Slack",
  "Slider",
  "SmallComponentsMakingALargerWhole",
  "Sneaker",
  "Snow",
  "Snowflake",
  "SocialWork_01",
  "SocialWork_02",
  "SocialWorker",
  "Socks",
  "SoftIceCream",
  "SoftlayerEnablement",
  "SolarField",
  "SolarPanel",
  "Solve",
  "Spaceship",
  "Speech",
  "SpeechToText",
  "Speedometer",
  "Sports",
  "Sprout",
  "StackLimitation",
  "Stairs",
  "StairsDown",
  "StairsPlanView",
  "StairsUp",
  "StartForFree",
  "StationaryBicycle",
  "Steel",
  "SteeringWheel",
  "StemLeafPlot",
  "Stethoscope",
  "Stockholm",
  "StorageAreaNetworks",
  "StorageProduct",
  "Strategy",
  "StrategyAndRisk",
  "StrategyDirect",
  "StrategyMove",
  "StrategyPlay",
  "StreamingData",
  "Streamline",
  "StreamlineOperations",
  "Subsecond",
  "Summit",
  "Sunny",
  "SunnyHazy",
  "SupplyChainOptimization_01",
  "SupplyChainOptimization_02",
  "SupplyChain_01",
  "SupplyChain_02",
  "Support",
  "SupportServices",
  "Sustainability",
  "SustainabilityStrategy",
  "Sustainability_02",
  "Sustainability_03",
  "Sustainability_04",
  "SwipeLeft",
  "SwipeRight",
  "Synergy",
  "SystemsDevopsAnalyze",
  "SystemsDevopsBuild",
  "SystemsDevopsCicdPipeline",
  "SystemsDevopsCode",
  "SystemsDevopsDeploy",
  "SystemsDevopsMonitor",
  "SystemsDevopsPlan",
  "SystemsDevopsProvision",
  "SystemsDevopsRelease",
  "SystemsDevopsTest",
  "TShirt",
  "TabletDevice",
  "TabletDeviceCheck",
  "Tags",
  "TaipeiBubbleTea",
  "TapeStorage",
  "Target",
  "TargetArea",
  "Teacher",
  "TeamAlignment",
  "Teammates",
  "TechnicalOwner",
  "TelAviv",
  "Telecom",
  "Telemedicine",
  "TelemedicineMobile",
  "Telephone",
  "Television",
  "TemperatureHigh",
  "TemperatureLow",
  "TemporaryBadge",
  "Tennis",
  "TestTubes",
  "TextData",
  "TextEquivalent",
  "TextInput",
  "TextLayout",
  "TextToSpeech",
  "ThisSideUp",
  "ThreatManagement",
  "Time",
  "TimeLapse",
  "TimePlot",
  "Toggle",
  "TokyoCherryBlossom",
  "TokyoGates",
  "TokyoTemple",
  "TokyoVolcano",
  "ToneAnalyzer",
  "ToolOverload",
  "Tools",
  "Tornado",
  "Toronto",
  "Touch",
  "TouchGesture",
  "TouchId",
  "TouchScreen",
  "TouchSwipe",
  "Tractor",
  "TradeoffAnalytics",
  "Train",
  "Training",
  "TransactionalBlockchain",
  "TransactionalTrust",
  "TransformData",
  "Transform_01",
  "Transform_02",
  "Transparency_01",
  "Transparency_02",
  "Trash",
  "Tree",
  "TreeDiagram",
  "TreeMap",
  "Trophy",
  "Troubleshooting",
  "Trousers",
  "Trust",
  "Trusted",
  "TrustedUser",
  "TwoPersonLift",
  "UnauthorizedUserAccess",
  "UnderUtilizedSecurity",
  "UnifyEndpointManagement",
  "UniversalExperiences",
  "University",
  "Unlock_01",
  "Unlock_02",
  "UnstructuredData",
  "Upload_01",
  "Upload_02",
  "UruguayPalacioSalvo",
  "UruguaySolDeMayo",
  "UseTheLanguageOfYourChoice",
  "User",
  "UserAnalytics",
  "UserExperienceDesign",
  "UserInsights",
  "UserInterface",
  "UserMask",
  "UserProfile",
  "UserSearch",
  "Vancouver",
  "VenezuelaNationalPantheonOfVenezuela",
  "VideoChat",
  "VideoPlay",
  "Video_01",
  "Video_02",
  "ViewGraphsAndDashboard",
  "VirtualServer",
  "VirtualStorage",
  "Virtualization",
  "Virtualization_02",
  "Virus",
  "Visibility",
  "Vision",
  "Visionary",
  "VisualData",
  "VisualDesign",
  "VisualInsights",
  "VisualRecognition",
  "Warning_01",
  "Warning_02",
  "Washer",
  "WashingtonDc",
  "WashingtonDcCapitol",
  "WashingtonDcMonument",
  "WatsonLogo",
  "Watsonx",
  "WatsonxAi",
  "WatsonxData",
  "WatsonxGovernance",
  "Weather",
  "WebDeveloper",
  "Webcast",
  "Websites",
  "WeddingCake",
  "Wheat",
  "Wifi",
  "WindPower",
  "Windows",
  "Windy",
  "Wine",
  "WirelessHome",
  "WirelessModem",
  "WordCloud",
  "Workday",
  "Workflows",
  "WorldCommunityGrid",
  "WreckingBall",
  "XRay_01",
  "XRay_02",
  "Yoga_01",
  "Yoga_02",
  "Yoga_03",
  "Yoga_04",
  "ZeroTrust",
]
`;
